### ISSUE TITLE
kubectl describe serviceaccount have redundance null line, we should keep accordance for kubectl describe command

### ISSUE THREAD

**Issue by zjj2wry** [kubernetes/kubernetes#52244]
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line: 
>
/kind bug
> /kind feature

**What happened**:
```
admindeMacBook-Pro-2:kubectl admin$ kubectl describe sa admin
Name:         admin
Namespace:    default
Labels:       <none>
Annotations:  <none>

Image pull secrets:  <none>

Mountable secrets:   admin-token-drw9x

Tokens:              admin-token-drw9x

Events:  <none>
```

**What you expected to happen**:
```
admindeMacBook-Pro-2:kubectl admin$ ./kubectl describe sa admin
Name:                admin
Namespace:           default
Labels:              <none>
Annotations:         <none>
Image pull secrets:  <none>
Mountable secrets:   admin-token-drw9x
Tokens:              admin-token-drw9x
Events:              <none>
```
**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration**:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by zjj2wry** [comment]
who can help remove the ```feature``` label

**Comment by zjj2wry** [comment]
Fixed by #52245
